Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -163,9 +165,10 @@ iOS applications often make use of third party libraries. These third party libr
 - Libraries that are not (or should not) be packed within the actual production application, such as `OHHTTPStubs` used for testing.
 - Libraries that are packed within the actual production application, such as `Alamofire`.
 
-These libraries can have the following two classes of unwanted side-effects:
+These libraries can lead to unwanted side-effects:
 
 - A library can contain a vulnerability, which will make the application vulnerable. A good example is `AFNetworking` version 2.5.1, which contained a bug that disabled certificate validation. This vulnerability would allow attackers to execute man-in-the-middle attacks against apps that are using the library to connect to their APIs.
+- A library can no longer be maintained or hardly be used, which is why no vulnerabilities are reported and/or fixed. This can lead to having bad and/or vulnerable code in your application through the library.
```

Comment:
Slightly re-phrased. What do you think? 

```suggestion
- A library is no longer maintained or is not widely used, which is why no vulnerabilities are reported and/or fixed. This can lead to having bad and/or vulnerable code in your application because of that library.
```